pom.xml

```diff
@@ -92,6 +93,7 @@
         <dep.plugin.checkstyle.version>3.0.0</dep.plugin.checkstyle.version>
         <dep.plugin.pmd.version>3.9.0</dep.plugin.pmd.version>
         <dep.plugin.sortpom.version>2.8.0</dep.plugin.sortpom.version>
+        <dep.plugin.spotbugs.version>3.1.10</dep.plugin.spotbugs.version>
```

Comment:
just a collateral improvement?

Reply:
Yeah, I tripped over some spotbugs issue and a version upgrade fixed it.  We were on a pretty old version.  I could make a separate PR if you like but it's literally this one line change.